superset/assets/src/explore/controls.jsx

```diff
@@ -324,6 +324,40 @@ export const controls = {
       ['red_yellow_blue', 'red/yellowish/blue'],
       ['brown_white_green', 'brown/white/green'],
       ['purple_white_green', 'purple/white/green'],
+      ['schemeBrBG', 'd3BrBG'],
```

Comment:
can you please make these more human readable? non-power users are not going to have any idea what `BrBG` means

Reply:
sure. updated.